packages/image-ui/src/ImageElement/ImageElement.tsx

```diff
@@ -64,50 +81,53 @@ export const ImageElement = (props: ImageElementProps) => {
       <div contentEditable={false}>
         <figure css={styles.figure?.css} className={`group ${styles.figure?.className}`}>
           <div css={styles.optionsToolbar.css} className={styles.optionsToolbar.className}>
-            <button
-              css={styles.optionsToolbarButton?.css}
-              className={styles.optionsToolbarButton?.className}
-              onClick={() => setSize(ImageSizeType.large)}>
-              lg
-            </button>
-            <button
-              css={styles.optionsToolbarButton?.css}
-              className={styles.optionsToolbarButton?.className}
-              onClick={() => setSize(ImageSizeType.medium)}>
-              md
-            </button>
-            <button
-              css={styles.optionsToolbarButton?.css}
-              className={styles.optionsToolbarButton?.className}
-              onClick={() => setSize(ImageSizeType.small)}>
-              sm
-            </button>
+            <ImageSizeButton
+              type={ImageSizeType.fullScreen}
+              label="screen"
+              onClick={setSize}
+              styles={styles.optionsToolbarButton}
+            />
+            <ImageSizeButton
+              type={ImageSizeType.large}
+              label="lg"
+              onClick={setSize}
+              styles={styles.optionsToolbarButton}
+            />
+            <ImageSizeButton
+              type={ImageSizeType.medium}
+              label="md"
+              onClick={setSize}
+              styles={styles.optionsToolbarButton}
+            />
+            <ImageSizeButton
+              type={ImageSizeType.small}
+              label="sm"
+              onClick={setSize}
+              styles={styles.optionsToolbarButton}
+            />
+          </div>
+          <div style={{width: imageSizeMap[imageSize]}}>
+            <img
+              data-testid="ImageElementImage"
+              css={styles.img?.css}
+              className={styles.img?.className}
+              src={url}
+              alt={captionString}
+              draggable={draggable}
+              {...nodeProps}
+            />
+            {!caption.disabled && (captionString.length || selected) && (
```

Comment:
The caption should also be richText

Reply:
@roosnic1 it is yes, but it doesn't have the functionality for bold italic and alignment for now.